development/nautobot_config.py

```diff
@@ -169,6 +169,7 @@
         "enable_intended": is_truthy(os.environ.get("ENABLE_INTENDED", True)),
         "enable_sotagg": is_truthy(os.environ.get("ENABLE_SOTAGG", True)),
         "sot_agg_transposer": os.environ.get("SOT_AGG_TRANSPOSER"),
+        "enable_postprocessing": is_truthy(os.environ.get("ENABLE_POSTPROCESSING", True)),
```

Comment:
Should a commented out `config_postprocessing_callables` be here?

Reply:
resolved in https://github.com/nautobot/nautobot-plugin-golden-config/commit/5b88c8afa091542b049f6de527ecb0d4001df50b